packages/button/components/StandardButton.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import { cx } from "emotion";
+import {
+  default as ButtonBase,
+  ButtonProps,
+  ButtonAppearances
+} from "./ButtonBase";
+import {
+  greyLightDarken1,
+  white
+} from "../../design-tokens/build/js/designTokens";
+import { focusStyles } from "../style";
+import { hexToRgbA } from "../../shared/styles/color";
+
+const StandardButton = (props: ButtonProps) => (
```

Comment:
How do you feel about omitting the `Standard`?  Shouldn't this just be ButtonBase?

Reply:
The design team calls it "Standard", so I'd rather we adopt their language.

Reply:
✅